pyproject.toml

```diff
@@ -58,7 +58,7 @@ tests = [
   "langchain_community",
   "pandas",
   "psycopg2",
-  "pydantic",
+  "pydantic[email]",
```

Comment:
email?

Reply:
test

Reply:
Elijah said he added it only for testing purposes :welp: